XCUITests/SaveLoginsTests.swift

```diff
@@ -5,36 +5,95 @@
 import XCTest
 
 let domain = "http://localhost:\(serverPort)"
+let domainLogin = "test@example.com"
+let domainSecondLogin = "test2@example.com"
 let testLoginPage = path(forTestPage: "test-password.html")
+let testSecondLoginPage = path(forTestPage: "test-password-2.html")
 let savedLoginEntry = "test@example.com, http://localhost:\(serverPort)"
 let urlLogin = "linkedin.com"
 let mailLogin = "iosmztest@mailinator.com"
+//The following seem to be labels that change a lot and make the tests break; aka volatile. Let's keep them in one place.
+let loginsListURLLabel = "Website, \(domain)"
+let loginsListUsernameLabel = "Username, test@example.com"
```

Comment:
There should be IDs for these elements, for example for [usernameField](https://github.com/mozilla-mobile/firefox-ios/blob/75f96daac75dd1c8a6583a481ec266acf0100d41/Client/Frontend/Settings/LoginDetailViewController.swift#L145) but somehow, they are not shown...could you please check why?